shared/login/reset/waiting.tsx

```diff
@@ -2,61 +2,78 @@ import * as React from 'react'
-        color={Styles.globalColors.black}
-        fontSize={24}
-      />
-      <Kb.Box2 direction="vertical" centerChildren={true} gap="small">
-        <Kb.Text type="Header" center={true}>
-          {props.pipelineStarted ? `Check back in ${props.time}` : 'Check your email or phone.'}
-        </Kb.Text>
-        {props.pipelineStarted ? (
-          <Kb.Box2 direction="vertical" centerChildren={true}>
-            <Kb.Text type="Body" style={styles.mainText} center={true}>
-              The reset has been initiated. For security reasons, nothing will happen in the next 7 days. We
-              will notify you once you can proceed with the reset.
-            </Kb.Text>
-            <Kb.Text type="Body">Unless you would like to</Kb.Text>
-            <Kb.Text type="BodyPrimaryLink" onClick={todo}>
-              cancel the reset.
-            </Kb.Text>
-          </Kb.Box2>
-        ) : (
-          <Kb.Box2 direction="vertical" centerChildren={true}>
-            <Kb.Text type="Body" style={styles.mainText} center={true}>
-              We are sending instructions to your email address or phone number.
-            </Kb.Text>
-            <Kb.Text type="BodyPrimaryLink" onClick={todo}>
-              Send again
-            </Kb.Text>
-          </Kb.Box2>
-        )}
+const Waiting = (props: Props) => {
+  const pipelineStarted = Container.getRouteProps(props, 'pipelineStarted', false)
+  const endTime = Container.useSelector(state => state.autoreset.endTime)
+
+  const dispatch = Container.useDispatch()
+  const nav = Container.useSafeNavigation()
+  const onClose = React.useCallback(
+    () => dispatch(nav.safeNavigateAppendPayload({path: ['login'], replace: true})),
+    []
+  )
+
+  return (
+    <SignupScreen
+      title="Account reset"
+      noBackground={true}
+      buttons={[{label: 'Close', onClick: onClose, type: 'Dim'}]}
+    >
+      <Kb.Box2 direction="vertical" gap="medium" fullWidth={true} fullHeight={true} centerChildren={true}>
+        <Kb.Icon
+          type={pipelineStarted ? 'iconfont-wave-2' : 'iconfont-mailbox'}
+          color={Styles.globalColors.black}
+          fontSize={24}
+        />
+        <Kb.Box2 direction="vertical" centerChildren={true} gap="small">
+          <Kb.Text type="Header" center={true}>
+            {pipelineStarted ? `Check back in ${formatDuration(endTime)}` : 'Check your email or phone.'}
+          </Kb.Text>
+          {pipelineStarted ? (
+            <Kb.Box2 direction="vertical" centerChildren={true}>
+              <Kb.Text type="Body" style={styles.mainText} center={true}>
+                The reset has been initiated. For security reasons, nothing will happen in the next{' '}
+                {formatDuration(endTime)}. We will notify you once you can proceed with the reset.
+              </Kb.Text>
+              <Kb.Text type="Body">Unless you would like to</Kb.Text>
+              <Kb.Text type="BodyPrimaryLink" onClick={todo}>
```

Comment:
the other integration branch has`cancelReset` plumbed, will ad this when that's merged